.github/workflows/docs.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish API documentation
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+*
+
+permissions: {}
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332  # tag: v4.1.7
+      - uses: actions/setup-node@1e60f620b9541d16bece96c5465dc8ee9832be0b  # tag: v4.0.3
+        with:
+          node-version: lts/*
+      - name: Install dependencies
+        run: yarn --frozen-lockfile
+      - name: Build API documentation
+        run: yarn build:docs
+      - name: Upload to Azure Blob Storage
+        uses: azure/cli@089eac9d8cc39f5d003e94f8b65efc51076c9cbd  # tag: v2.1.0
+        with:
+          azcliversion: latest
+          inlineScript: |
+            az storage blob upload-batch --account-name electronecosystemapidocs -d '$web/notarize/${{ github.ref_name }}' -s ./docs --overwrite --sas-token "$SAS_TOKEN"
```

Comment:
Could we move the account-name variable into a secret, just to obfuscate it a bit?

Reply:
[29a7845](https://github.com/electron/notarize/pull/217/commits/29a784529bc2b333352a160ab4e9bf97532c0313)